canonical-web-and-design/ubuntu.com
# redirect shop links to buy.u.c

## Done

Redirected /advantage/subscribe and /advantage/subscribe/thank-you to buy.ubuntu.com while shop launch is blocked, to unblock u.c deploys.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- If you're logged in, click the "Add another subscription" dropdown and the "Buy a subscription" button within, see that you are redirected to buy.ubuntu.com
- If you're logged out, click the "Buy a subscription" button in the hero strip, see that you are redirected to buy.ubuntu.com
